CMakeLists.txt

```diff
@@ -13,11 +13,19 @@ include(GNUInstallDirs OPTIONAL)
 add_definitions(-DINSTALL_BINDIR="${CMAKE_INSTALL_BINDIR}")
 add_definitions(-DINSTALL_DATADIR="${CMAKE_INSTALL_DATADIR}")
 
+if(WIN32)
+  set(USE_INTERNAL_LIBS 1)
+else()
+  set(USE_INTERNAL_LIBS 0)
+endif()
+
 # options
 option(BUILD_CLIENT "Build client target" 1)
 option(BUILD_SERVER "Build server target" 1)
 option(BUILD_MASTER "Build master server target" 1)
 option(BUILD_ODALAUNCH "Build odalaunch target" 1)
+cmake_dependent_option( USE_INTERNAL_ZLIB "Use internal zlib" ${USE_INTERNAL_LIBS} BUILD_CLIENT 0 )

```

Comment:
Based on my reading of cmake_dependent_option the USE_INTERNAL_LIBS variable can be overridden by the user if they configure with USE_INTERNAL_ZLIB and USE_INTERNAL_PNG set to true. Assuming that is correct I think this is fine for WIN32 build.